### DISCUSSION TITLE
Can we use Turbopack with a custom server?

### DISCUSSION THREAD

**Discussion by mdj-uk** [vercel/next.js#49325] [Help]
### Summary

According to the docs turbopack is now in beta and can be used in development with `next dev --turbo`.

However I'm using next with a custom express server, following the pattern in the example in  [the docs](https://nextjs.org/docs/pages/building-your-application/configuring/custom-server).  So we don't explicitly call `next dev`, instead my package.json contains
```json
"scripts": {
  "dev": "cross-env NODE_ENV=development nodemon index.js"
}
```
Is there any way to use turbopack in this case?

**Comment by neotrow** [comment]
I am also interested in an answer :-)

**Comment by KennyStanleyJr** [comment]
Same here!

**Comment by the-digital-anarchist** [comment]
Did any one got this working? Bin searching for hours now

**Comment by zetoke** [comment]
Yeah, i've been looking for this one as well. 
And looks like the answer is no. 

I want to try it out in development but seems like it only supports it running from CLI without a custom server.

**Comment by stychu** [comment]
+1 Want to try it too, looks we are abandoned and no chance to try this way sadge..

**Comment by andreasasprou** [comment]
Try setting the environment variable `TURBOPACK="1"` 

https://github.com/vercel/next.js/blob/b9dd6c4ff6b025453504c6a5669094260c23ca1b/packages/next/src/cli/next-dev.ts#L213

**Reply by mjozan** [reply]
it does not work for me

**Comment by jamesopti** [comment]
Any updates here with the release of Next14?

**Comment by drewhamlett** [comment]
`TURBOPACK=1 NODE_ENV=development tsx watch src/server.ts`
 
Works for me (Next 14.0.4)
